Engine/linevent.c

```diff
@@ -575,6 +589,17 @@ int32_t eventOpcodeI_(CSOUND *csound, LINEVENT *p, int32_t insname, char p1)
         for (i = 2; i <= evt.pcnt; i++)
            evt.p[i] = *p->args[i];
       }
+      if (insname == 2) {
```

Comment:
```suggestion
      else if (insname == 2) {
```

Seems like this should be `else if` here, in case insname == 1 so that the following else doesn't trigger

Reply:
done